README.md

```diff
@@ -53,125 +60,8 @@ cd ao
-# convert linear modules to quantized tensor subclasses
-quant_api.change_linear_weights_to_int8_dqtensors(model)
-```
-
-### A16W8 WeightOnly Quantization
-
-```python
-from torchao.quantization import quant_api
-quant_api.change_linear_weights_to_int8_woqtensors(model)
-```
-
-This technique works best when the torch._inductor.config.use_mixed_mm option is enabled. This avoids dequantizing the weight tensor before the matmul, instead fusing the dequantization into the matmul, thereby avoiding materialization of a large floating point weight tensor.
-
-
-### A16W4 WeightOnly Quantization
-
-```python
-from torchao.quantization import quant_api
-quant_api.change_linear_weights_to_int4_woqtensors(model)
-```
-
-Note: The quantization error incurred by applying int4 quantization to your model can be fairly significant, so using external techniques like GPTQ may be necessary to obtain a usable model.
-
-
-### A8W8 Dynamic Quantization with Smoothquant
-
-We've also implemented a version of [smoothquant](https://arxiv.org/abs/2211.10438) with the same GEMM format as above. Due to requiring calibration, the API is more complicated.
-
-Example
-
-```Python
-import torch
-from torchao.quantization.smoothquant import swap_linear_with_smooth_fq_linear, smooth_fq_linear_to_inference
-
-# Fuse the int8*int8 -> int32 matmul and subsequent mul op avoiding materialization of the int32 intermediary tensor
-torch._inductor.config.force_fuse_int_mm_with_mul = True
-
-# plug in your model
-model = get_model()
-
-# convert linear modules to smoothquant
-# linear module in calibration mode
-swap_linear_with_smooth_fq_linear(model)
-
-# Create a data loader for calibration
-calibration_data = get_calibration_data()
-calibration_dataset = MyDataset(calibration_data)
-calibration_loader = DataLoader(calibration_dataset, batch_size=32, shuffle=True)
-
-# Calibrate the model
-model.train()
-for batch in calibration_loader:
-    inputs = batch
-    model(inputs)
-
-# set it to inference mode
-smooth_fq_linear_to_inference(model)
-
-# compile the model to improve performance
-model = torch.compile(model, mode='max-autotune')
-model(input)
-```
-
-
-## Notes
-
-1. APIs have been hardware tested on A100 and T4(colab) 
-2. While these techniques are designed to improve model performance, in some cases the opposite can occur. This is because quantization adds additional overhead to the model that is hopefully made up for by faster matmuls (dynamic quantization) or loading weights faster (weight-only quantization). If your matmuls are small enough or your non-quantized perf isn't bottlenecked by weight load time, these techniques may reduce performance.
-3. Use the PyTorch nightlies so you can leverage [tensor subclasses](https://pytorch.org/docs/stable/notes/extending.html#subclassing-torch-tensor) which is preferred over older module swap based methods because it doesn't modify the graph and is generally more composable and flexible.
-
+## Get Started
```

Comment:
@msaroufim @supriyar I added a get started section here and linked to the API READEMEs